pkg/cli/options.go

```diff
@@ -1,68 +1,57 @@
 package cli
 
 import (
+	"flag"
 	"io"
-	"io/ioutil"
 	"os"
 
-	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
-	"golang.org/x/crypto/ssh/terminal"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/klog/v2"
 )
 
 type RootOptions struct {
 	genericclioptions.IOStreams
 
 	Dir      string
-	LogLevel string
+	LogLevel int
 
 	logfileCleanup func()
 }
 
 func (o *RootOptions) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVarP(&o.Dir, "dir", "d", "oc-mirror-workspace", "Assets directory")
-	fs.StringVar(&o.LogLevel, "log-level", "info", "Log level (e.g. \"debug | info | warn | error\")")
+	//TODO tell int is needed vs str now
+	fs.IntVarP(&o.LogLevel, "verbose", "v", 2, "Log level (e.g. \"Error (1), Info (2) | Warning (3) | Debug (4)\")")
 	if err := fs.MarkHidden("dir"); err != nil {
-		logrus.Panic(err.Error())
+		klog.Fatal(err.Error())
 	}
 }
 
 func (o *RootOptions) LogfilePreRun(cmd *cobra.Command, _ []string) {
-	logrus.SetOutput(ioutil.Discard)
-	logrus.SetLevel(logrus.TraceLevel)
 
-	level, err := logrus.ParseLevel(o.LogLevel)
+	var fsv2 flag.FlagSet
+	klog.InitFlags(&fsv2)
+	checkErr(fsv2.Set("stderrthreshold", "4"))
+
+	logFile, err := os.OpenFile(".oc-mirror.log", os.O_CREATE|os.O_APPEND|os.O_RDWR, 0666)
 	if err != nil {
-		logrus.Fatalf("parse root options log-level: %v", err)
+		panic(err)
```

Comment:
@sfxworks I think `go vet` is complaining here. Can you use klog.Fatal instead of `panic`?